### DISCUSSION TITLE
Payment Failed در درگاه مجازی و خطای Invalid data is received from the gateway در درگاه ملت

### DISCUSSION THREAD

**Discussion by Asp1984** [Sina-Soltani/Parbad#282] [Q&A]
با سلام و خسته نباشید
در زمان استفاده روی لوکال و اتصال به درگاه مجازی در زمان اجرای 
`var verifyResult = await _onlinePayment.VerifyAsync(invoice);`
بازگشتی متد 
Payment Failed
هستش و در زمان اجرای پروژه روی هاست و اتصال به درگاه ملت : اتصال به درگاه انجام  میشه و نتیجه پرداختی نیز بدرستی بازگشت داده میشه به عنوان نمونه به شرح ذیل:
`{"Status":0,"IsAlreadyVerified":false,"IsSucceed":true,"TrackingNumber":68,"Amount":{"Value":200000.00},"GatewayName":"Mellat","GatewayAccountName":"Default","Message":null,"AdditionalData":{},"DatabaseAdditionalData":{}}`

ولی باز در زمان اجرای متد 
`var verifyResult = await _onlinePayment.VerifyAsync(invoice);`
به مشکل برمی خورم و پاسخ بانک میشه

 Invalid data is received from the gateway 
 
 ممنون میشم راهنمایی بفرمایید

**Comment by Sina-Soltani** [comment]
درود
یک تست جدید برای درگاه مجازی انجام شد. بدون ایراد کار میکنه.

![image](https://user-images.githubusercontent.com/10329804/153751778-55cbf8cd-9552-41fc-89e3-e0dfd431f2f5.png)

لطفا تنظیمات رو اینجا قرار بدید تا بررسی کنیم مشکلتون رو (بدون ذکر اطلاعات حساس)
